### ISSUE TITLE
Cannot restart a task with the interactive client

### ISSUE THREAD

**Issue by sandrineBeauche** [ow2-proactive/scheduling#2289]

**Comment by lpellegr** [comment]
Based on the source code, it seems that a task can be restarted if it is in one of the following states:

  - SUBMITTED
  - PENDING
  - PAUSED
  - RUNNING
  - WAITING_ON_ERROR
  - WAITING_ON_FAILURE

I tried the RUNNING state with a Python task sleeping for 60 seconds and with `maxNumberOfExecution` set to at least 2:

```python
import time
time.sleep(60)
```

In that case a restart with `./bin/proactive-client -rt 1010 "Python_Task"` seems to work.

If the `maxNumberOfExecution` is set to a lower value, then `proactive-client` reports that the restart has succeeded but the task fails.

The same test for restarting a task in a PAUSED state seems not to work.

Also, the proactive-client help command speaks about a *delay* parameter that cannot be set using the utility.

To summarize, the usage is not well documented and some cases seem not to work.
